jsconfig.json

```diff
@@ -2,5 +2,8 @@
   "include": [
     "./app/javascript/**/*"
   ],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "vueCompilerOptions": {
```

Comment:
what is the purpose of this section?  to explicitly target vue2 instead of vue3?  why was this not needed earlier?